src/helper_classes/HttpHeader.java

```diff
@@ -0,0 +1,40 @@
+package helper_classes;
```

Comment:
I am checking with Lee about package name for our library. In any case please follow the style guide. Refer https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

Reply:
done